css/index.css

```diff
@@ -112,6 +136,47 @@ p {
     justify-content: center;
 }
 
```

Comment:
Great use of ids since they are only used one time on this page.  Good job overriding only the color in each of the individual boxs and letting the parent do the rest of the styling.